ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1249,6 +1249,24 @@ FCU_CHWZTC2X_HT2XZTC2X_DFVSC2X:
   - CHWZTC2X
   - HT2XZTC2X
   - DFVSC2X
+  
+FCU_DFVSMC_CHWBRC_STM_RTM_FDPM_ZTC_SRC:
+  description: "FCU with Status, zone control, schedule run"
+  is_canonical: false
+  implements:
```

Comment:
please also add `DFSS` and rename the type accordingly.

Reply:
noted sir. Thanks

Reply:
Not sure about chilled water sir. It is Pre-Cooled Fan Coil Units (Ceiling Concealed Ducted Type) but here is where it is confusing me too: it has 2 point list for zones that is described as "Zone Air Temperature", "Zone Temperature Set Point Adjustment/Control". the 1st one is sensor, but the 2nd one is a combined setpoint and command. so, CHWZTC for me is not an option(?) as it has 3 points for zone. I'm not sure what to do here.

Reply:
It seems like your fieldset is incomplete. The abstract types I specified above seem to be correct, but this particular device doesn't provide you with chilled water valve control directly (it does do chilled water cooling, you just don't have points for the valve). This is an example of an incomplete translation. In your building config, you would specify that it is this type (including CHWZTC) but you would mark those specific fields as `MISSING`. Does that make sense? You can see an example of it [here](https://github.com/google/digitalbuildings/blob/6e597b34556d71d9c9c1f1f835819526601b5da0/ontology/docs/building_config.md#defining-translations).

Reply:
I see sir. Ok, I can do that. will adjust now to what you recommend, thanks sir

Reply:
please resolve this comment when you have made the changes

Reply:
will do sir. this is what we came up with. we can't use CHW and these below are the closest to waht we need, with additional points that will be marked MISSING. thanks sir
```
FCU_RTM_DXZTC_SRC:
  description: "FCU with SS, supply and return temp sensors, zone control and sensor, schedule run, filter change and general fail alarm"
  is_canonical: true
  implements:
  - FCU
  - RTM
  - DXZTC
  - SRC
  uses:
  - filter_alarm
  - lost_power_alarm
```